scripts/bootstrap

```diff
@@ -54,6 +54,10 @@ function pull_sbtopts() {
     sort app-backend/.sbtopts | uniq > cleaned_sbtopts
     mv cleaned_sbtopts app-backend/.sbtopts
 
+    git config --local filter.sbtopts.clean \
```

Comment:
Perhaps getting this to work with Git is an uphill battle that isn't worth continuing to fight. As an alternative, maybe we can add `.sbtopts` to ignore completely, but then use `bootstrap` to render its contents.

If `bootstrap` can successfully resolve DNS for the development Nexus instance, set the appropriate `repositories` file. Same if it can resolve the CI Nexus instance. If it can't resolve either, it sets the contents to:

```
-Dsbt.boot.directory=project/.boot
-Dsbt.global.base=project/.sbtboot
-Dsbt.ivy.home=project/.ivy
```

Setting the contents to the above could also be achieved by having a `.sbtopts.sample` file that we `mv` into place.